src/core/src/osc_server.cpp

```diff
@@ -822,7 +827,7 @@ void OscServer::start()
 	m_pServerThread->start();
 
 
-	INFOLOG(QString("Osc server started. Listening on port %1").arg( m_pPreferences->getOscServerPort() ));
+	INFOLOG(QString("Osc server started. Listening on port %1").arg( m_pPreferences->get_instance()->getOscServerPort() ));
```

Comment:
Is this intentional?

Reply:
Nope, that one slipped through. Just pushed a fix